ocs_ci/ocs/ui/page_objects/virtualmachine_ui.py

```
@@ -0,0 +1,405 @@
+        checked = False
+        try:
+            els = self.driver.find_elements(By.XPATH, grace_xpath)
+            if els:
+                if not els[0].is_selected():
+                    els[0].click()
+                    logger.info("Checked 'With grace period'")
+                else:
+                    logger.info("'With grace period' already checked")
+                checked = True
+        except Exception:
+            pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)